app/models/fixed_session.rb

```diff
@@ -24,6 +24,22 @@ def self.streaming
     where("last_measurement_at > ?", Time.current - 1.hour)
   end
 
+  def self.dormant
+    where("last_measurement_at <= ?", Time.current - 1.hour)
+  end
+
+  def self.all_dormant(data, limit, offset)
```

Comment:
sorry, I mean that `all_dormant` right now doesn't really return all dormant sessions but rather a specific batch, while `dormant` returns all dormant sessions. I think that it might make more sense to have it the other way around, unless you have a better idea?

Reply:
I see what you mean. I'm not gonna do that at the moment cause that `dormant` is the symmetrical counter part of `streaming`. Also, I'm planning to make a project wide rename of `streaming` -> `active` and the same for `dormant`. Will try to fit this change there